templates/full-stack.template.yaml

```
@@ -63,7 +63,7 @@ Parameters:
   BuildImageName:
     Description: Docker image for application build
     Type: String
-    Default: aws/codebuild/nodejs:10.1.0
+    Default: lambci/lambda:build-nodejs12.x	
```

```suggestion
    Default: aws/codebuild/amazonlinux2-x86_64-standard:2.0	
```

Then specify the `nodejs: 12` runtime in the buildspec.yml as shown in https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html#runtime-versions-buildspec-file

---

Thanks, added in https://github.com/aws-quickstart/quickstart-trek10-serverless-enterprise-cicd/pull/22/commits/8d96e818f8535be87380d91afb3e613d927fc3db